.ci/jobs/apm-integration-tests-mbp-jjbb.yml

```diff
@@ -0,0 +1,45 @@
+---
+- job:
+    name: apm-integration-tests-mbp-jjbb
+    display-name: APM Integration Test MBP
```

Comment:
on display name, we should put something to differentiate them from the currents until we remove them

Reply:
We are adding `MBP` after old name